modules/builtin/src/translations/en.json

```diff
@@ -62,7 +62,7 @@
     "text": {
       "add": "Add Text Alternatives",
       "alternative": "Alternative",
-      "alternatives": "Alternatives",
+      "alternative_plural": "Alternatives",
```

Comment:
this is now what I meant but okay this is fine

Reply:
It's how react-i18n works apparently, you can pass count: and it'll pluralize for you. I didn't try it, but it's supposed to work. The reason I didn't use the singular then passed a count was that it wouldn't make sense to fake a count just for the purpose of using a plural